src/platforms/dotnet/guides/xamarin/common/troubleshooting.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title:  Troubleshooting
+sidebar_order: 100
+description: "Solutions to common issues"
+---
+
+## Offline Caching Information 
+
```

Comment:
Going directly from one heading level to the next (without text between) interferes with the RH sidebar. Perhaps add something similar to:
"For users on Xamarin iOS, offline caching may cause this tricky issue."